### PR TITLE
Here are Some DS/Algo programs in .cpp 

### PR DESCRIPTION
**Pull request by mayank5464942**

### CODE REVIEW

**Comment by vanshikaarora** [comment]
I didn't see any changes. Have you pushed your commit?

**Comment by vanshikaarora** [comment]
Please push your changes, if you have done any

**Comment by mayank5464942** [comment]
Please review the changes

File: LinkedList/Swapping elements of LL without Swapping Data [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+	}
+	
+	int a,b;
+	cin>>a>>b;
+	
+	Node *h1;
+	h1 = swap(A.gethead(),a,b);
+	print_list(h1);
+	return 0;
+}
```
Comment (vanshikaarora) [inline LinkedList/Swapping elements of LL without Swapping Data]: Please change the extension of this file to ".cpp"

File: LinkedList/Linked List using Recursion [hunk 2]
```diff
@@ -0,0 +1,70 @@
+#include <iostream>
```
Comment (vanshikaarora) [inline LinkedList/Linked List using Recursion]: same for this file

Comment (mayank5464942) [reply LinkedList/Linked List using Recursion]: Extension changed to .cpp for both files

**Pull request merged**